packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -99,8 +99,8 @@ class VmServiceWrapper implements VmService {
 
   @override
   Future<Success> clearCpuProfile(String isolateId) {
-    return _trackFuture(
-        'clearCpuProfile', _vmService.clearCpuProfile(isolateId));
+    return _trackFuture('clearCpuProfile',
+        callMethod('_clearCpuProfile', isolateId: isolateId));
   }
```

Comment:
Is this backwards compatible with all possible versions of the vm_service users will be on?

Reply:
At this point, yes. No changes have been made to the `_clearCpuProfile` RPC. This change is just to allow for the removal of the `clearCpuProfile` method in `package:vm_service` as it's currently undocumented.

Reply:
So once we are ported to package:vm_service, will `clearCpuProfile` again become a public RPC? I guess I'm unclear why we are going back to calling the private RPC here.

Reply:
This was never a public RPC, so this is effectively a no-op from a functionality perspective. Maybe you're thinking of [clearVMTimeline](https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#clearvmtimeline)?